builder/main.py

```
@@ -515,6 +517,103 @@ def build_fs_image(target, source, env):
+        return 0
+
+    except Exception as e:
+        print(f"Error building FatFS image: {e}")
+        return 1
+
+        with open(target_file, "wb") as f:
+            f.write(storage)
+
+        return 0
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)